src/Pipfile

```diff
@@ -9,6 +9,9 @@ importlib-metadata = "*"
 pyahocorasick = "*"
 
 [dev-packages]
+tqdm = "*"
+pandas = "*"
```

Comment:
Pandas is a very big and heavy dependency. Do we actually need it? Can we avoid it?

Reply:
Yes i believe we can.